R/build.r

```diff
@@ -171,7 +171,7 @@
 #'
 #' You can also override the default templates and provide additional
 #' assets. You can do so by either storing in a `package` with
```

Comment:
I think this needs more tweaking — you can either put them in `pkgdown/...` in the current package, or you can create a template package. But maybe it's better to not mention the template package here, and instead just point to the customisation vignette.